### PR TITLE
Docs/update ecommerce integrations

### PR DESCRIPTION
**Pull request by kwiliwis**

### CODE REVIEW

File: docs/integrations/index.md [hunk 1]
```diff
@@ -1,9 +1,45 @@
...
+
+#### 1. **Google Merchant Center (Google-Shopping)**
+   - **URL**: [Google Shopping Feed in XML](https://addons.prestashop.com/en/price-comparison/1768-google-merchant-center-google-shopping.html)
+   - **Features**: Easy setup, custom feed templates, and multi-store support.
+
+## Best Practices for Integration
+
+1. **Consistent Updates**: Regularly update your product feed to ensure accurate pricing and availability information.
+2. **Optimize Product Data**: Include detailed product titles, descriptions, and high-quality images to improve visibility and click-through rates.
+3. **Leverage Analytics**: Monitor the performance of your listings on Prisjakt to optimize pricing strategies and promotions.
```
Comment (andrejohansson) [inline docs/integrations/index.md:45]: Nitpicky note: Prefer not numbering, just put 1. on all bullets and the markdown renderer will do it for you (per specification).

Comment (kwiliwis) [reply docs/integrations/index.md]: Fixed

File: docs/e-commerce-integrations.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- **Direct Traffic**: Drive potential customers directly to your product pages.
+
+## E-commerce Platform Plugins
+
+We've listed popular and free (or freemium) plugins for generating Prisjakt compatible product feeds in various e-commerce platforms.
+
+### Shopify
+
+#### 1. **Google Shopping Feed XML/CSV**
+   - **URL**: [Google Shopping Feed XML/CSV](https://apps.shopify.com/simple-google-shopping-feed)
```
Comment (andrejohansson) [inline docs/e-commerce-integrations.md:20]: Any specific reason for numbering the headlines?

Comment (kwiliwis) [reply docs/e-commerce-integrations.md]: Nope, just leftover. I removed it and fixed .md files formatting.

**Pull request merged**